### ISSUE TITLE
Update AWS maintenance docs

### ISSUE THREAD

**Issue by reshmabidikar** [killbill/killbill-docs#562]
The following changes need to be done to the AWS maintenance docs:

1. Add ELB related maintenance steps to the [multitier maintenance doc](https://docs.killbill.io/latest/how-to-maintain-a-multi-tier-system) if required
2. Add information about SSL Certificate to the  [multitier maintenance doc](https://docs.killbill.io/latest/how-to-maintain-a-multi-tier-system) if required
3. Add information about updating configuration parameters via the template to the [cf maintenance doc](https://docs.killbill.io/latest/how-to-maintain-a-cloud-formation-system)

**Comment by reshmabidikar** [comment]
For 3/ - These are the notes from @vnandwana:

```
On CF dashboard, there is an option to update a stack. For more info, please check https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks.html. Also, customers can definitely SSH to an EC2 instance and update properties manually.
```

**Comment by vnandwana** [comment]
This PR (https://github.com/killbill/killbill-docs/pull/584) adds the missing information about updating the CloudFormation stack.

@reshmabidikar, is there anything else pending for this task?

**Comment by reshmabidikar** [comment]
> This PR (#584) adds the missing information about updating the CloudFormation stack.
> 
> @reshmabidikar, is there anything else pending for this task?

Yes, 1/ and 2/ [above](https://github.com/killbill/killbill-docs/issues/562#issue-2444188469) are still pending.

**Comment by vnandwana** [comment]
Got it. Let me know if there’s anything I need to do or if you need my help.
